cmd/vela-email/plugin.go

```diff
@@ -141,6 +144,10 @@ func (p *Plugin) Validate() error {
 		return ErrorMissingSMTPParam
 	}
 
+	if len(p.Auth) > 0 && !(len(p.SMTPHost.Username) > 0 && len(p.SMTPHost.Username) > 0) {
```

Comment:
```suggestion
	if len(p.Auth) > 0 && !(len(p.SMTPHost.Username) > 0 && len(p.SMTPHost.Password) > 0) {
```

Reply:
yikes, sorry i missed that :shame_on_me: